opensoundscape/preprocess/actions.py

```diff
@@ -504,7 +504,7 @@ def overlay(
         ), "overlay_df must have labels if overlay_class is specified"
         if overlay_class != "different":  # user specified a single class
             assert (
-                np.sum(overlay_df[overlay_class]) > 0
+                sum(overlay_df[overlay_class]) > 0
```

Comment:
The title of the issue indicates that we should use pandas attributes on pandas Series. I don't really care which we use, but this uses native sum instead of the pandas Series.sum

Reply:
Have renamed it to say pandas/native to signify either, and used pandas one in this case. The code to set seeds for all random methods across packages is a set of 4-5 statements that we can include in the documentation.